opm/autodiff/GeoProps.hpp

```diff
@@ -224,6 +224,15 @@ namespace Opm
             }
         }
 
+        /// transmissibility values per face (Eigen::ArrayXd trans_) values
+        std::vector<double> face2trans() const {
```

Comment:
Since this method is only used from a single spot, I'd rather not add this method at all. Better to just do
```
const auto& trans = geoprops_->transmissibilities();
trans_.assign(trans.data(), trans.data() + trans.size());
```
The `col(0)` part is not necessary for an Eigen 1D Array.